src/System.Console/tests/NegativeTesting.cs

```diff
@@ -9,17 +9,33 @@
 public class NegativeTesting
 {
     [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
     public static void OpenStandardOutNegativeTests()
     {
         NegativeConsoleOutputTests(Console.OpenStandardOutput());
     }
 
     [Fact]
+    [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
```

Comment:
Same as above, I would change all the conditions to IsWinRT or IsNotWinRT now. I will stop commenting on others now.